<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DOS Games Manager</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>dosgamesicon.ico</normaloff>dosgamesicon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="box_art">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>141</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string>boxart</string>
    </property>
   </widget>
   <widget class="QPushButton" name="play_button">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>280</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Play selected game</string>
    </property>
   </widget>
   <widget class="QListWidget" name="games_list">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>341</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="add_game_button">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>270</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Game</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gamedetails">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>251</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Game details:</string>
    </property>
    <widget class="QPushButton" name="manual_button">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>150</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Open Manual</string>
     </property>
    </widget>
    <widget class="QLabel" name="completed_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Completed:</string>
     </property>
    </widget>
    <widget class="QLabel" name="developer_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Developer:</string>
     </property>
    </widget>
    <widget class="QLabel" name="publisher_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Publisher:</string>
     </property>
    </widget>
    <widget class="QLabel" name="year_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Year:</string>
     </property>
    </widget>
    <widget class="QLabel" name="genre_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Genre:</string>
     </property>
    </widget>
    <widget class="QLabel" name="playtime_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Play time:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="extras_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse Extras</string>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="hidecompleted_box">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>210</y>
      <width>191</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Hide completed games</string>
    </property>
   </widget>
   <widget class="QPushButton" name="defaultdos_button">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>320</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Launch default DOSBox to set up directories and games to be added.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Launch normal DOSBox</string>
    </property>
   </widget>
   <widget class="QPushButton" name="order_button">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>210</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Order by...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="edit_game_button">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>300</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="del_game_button">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>330</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Game</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
